Makefile

```diff
@@ -24,40 +24,6 @@ yarn_install_cache:  ## Use Yarn to install UI dependencies
 yarn_start_dev_ui:  ## Run the UI locally with Yarn
 	cd openmetadata-ui/src/main/resources/ui && yarn start
 
-## Ingestion Core
-.PHONY: core_install_dev
-core_install_dev:  ## Prepare a venv for the ingestion-core module
```

Comment:
cleaning up the ingestion-core since we're not really making any use of it. We can recover the files if we have the need in the future